cider/exec/module/CiderRuntimeModule.cpp

```diff
@@ -888,6 +889,40 @@ CiderBatch CiderRuntimeModule::setSchemaAndUpdateAggResIfNeed(
         }
       }
     }
+  } else if (ciderCompilationOption_.use_cider_data_format) {
```

Comment:
if (ciderCompilationOption_.use_cider_data_format && is_group_by_) would be better?